packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -93,6 +93,11 @@ export function getPluginOptions(
     telemetry: sourceMapsUploadOptions.telemetry ?? true,
     url: sourceMapsUploadOptions.url ?? process.env.SENTRY_URL,
     debug: moduleOptions.debug ?? false,
+    silent: sourceMapsUploadOptions.silent ?? false,
+    errorHandler: sourceMapsUploadOptions.errorHandler,
+    release: {
+      name: sourceMapsUploadOptions.release?.name,
+    },
```

Comment:
Also add the options from the `unstable_sentryBundlerPluginOptions` here :)
```suggestion
    release: {
      name: sourceMapsUploadOptions.release?.name,
      ...moduleOptions?.unstable_sentryBundlerPluginOptions?.release,
    },
```